pygments/lexers/configs.py

```diff
@@ -1127,3 +1127,99 @@ def analyse_text(text):
             result += 0.49
 
         return result
+
+
+class PasswdLexer(RegexLexer):
+    """
+    Lexer for /etc/passwd files.
+    """
+
+    name = 'PasswdLexer'
```

Comment:
The name should be the name of the language, not that of the lexer class. I'd use "/etc/passwd".